src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -123,6 +129,11 @@ def run(self) -> List[nodes.Node]:
         autoversion_mode = config.towncrier_draft_autoversion_mode
         include_empty = config.towncrier_draft_include_empty
 
+        for path in iter_towncrier_files(
+            fragments_dir=Path(config.towncrier_draft_working_directory)
```

Comment:
This part is inaccurate. `towncrier_draft_working_directory` is just a CWD of a subprocess, nothing more. Then, the process reads the config in `pyproject.toml` and that's where the correct directory is specified.

So the right implementation should read the toml config (+1 dependency!) and calculate the relative path to the current cwd (probably).

Reply:
Hm, or we could ask towncrier this info maybe. This would avoid the extra dep, but only if towncrier supports that.

Reply:
towncrier doesn't have any official public API other than CLI. And the CLI doesn't support that (well, maybe in log).

OTOH since you mentioned it, Towncrier already uses a TOML parser so it'll be available either way. We just have to make sure to rely on the same lib...